maxim/maxim.py

```
@@ -755,13 +755,13 @@ def _check_if_repo_exists(self, logger: Logger):
 
         def check():
             try:
-                exists = self.maxim_api.does_log_repository_exist(logger.repo_id)
+                exists = self.maxim_api.does_log_repository_exist(logger.id)
                 if not exists:
-                    scribe().warning(f"[MaximSDK] Log repository not found: {logger.repo_id}. We will be dropping all logs.")
+                    scribe().warning(f"[MaximSDK] Log repository not found: {logger.id}. We will be dropping all logs.")
                     if self.raise_exceptions:
-                        raise ValueError(f"Log repository not found: {logger.repo_id}")
+                        raise ValueError(f"Log repository not found: {logger.id}")
```

Ruff (0.11.9)

Abstract `raise` to an inner function

(TRY301)